expression/interpreter/interpreter.go

```diff
@@ -100,6 +102,31 @@ func (ei *ExprInterpreter) InterpretString(strRaw string) ([]byte, error) {
 		return []byte{}, nil
 	}
 
+	// file contents
+	// TODO: make this part of a proper parser
+	if strings.HasPrefix(strRaw, mxscPrefix) {
+		if ei.FileResolver == nil {
+			return []byte{}, errors.New("parser MxscResolver not provided")
+		}
+		fileContents, err := ei.FileResolver.ResolveFileValue(strRaw[len(mxscPrefix):])
+		if err != nil {
+			return []byte{}, err
+		}
+
+		mxsc := make(map[string]interface{})
```

Comment:
Can we extract this logic into a separate function? Just code extraction (string -> string)